GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -995,7 +995,7 @@ public void ForceRefreshRevisions()
                 _superprojectCurrentCheckout = null;
                 ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
                 {
-                    var scc = await GetSuperprojectCheckoutAsync(ShowRemoteRef, capturedModule, noLocks: true);
+                    var scc = await GetSuperprojectCheckoutAsync(capturedModule, noLocks: true);
```

Comment:
Can `RevisionGridControl.ShowRemoteRef` be removed, too?

Reply:
Yes, the method was used in one other occasion, that filtered the refs a second time